File: templates/python/en/text/code/stepUp.py
# File: FirstKarel.py
# -----------------------------
# The FirstKarel program defines a "main" 
# function with commands that cause Karel to pick up
# a beeper and place it on a ledge.
from karel.stanfordkarel import *

def main():
   move()
   pick_beeper()
   move()
   turn_left()
   move()
   turn_left()
   turn_left()
   turn_left()
   move()
   move()
   put_beeper()
   move()
   

File: templates/java/en/text/code/stepUp.java
/*
 * File: BeeperPickingKarel.java
 * -----------------------------
 * The BeeperPickingKarel program defines a "run" 
 * method with commands that cause Karel to pick up
 * a beeper and place it on a ledge.
 */
import stanford.karel.*;
public class BeeperPickingKarel extends Karel {
   public void run() {
      move();
      pickBeeper();
      move();
      turnLeft();
      move();
      turnLeft();
      turnLeft();
      turnLeft();
      move();
      move();
      putBeeper();
      move();
   } 
}